coding_introduction/CodingIntroduction.md

```diff
@@ -308,6 +373,33 @@ Regular files have filemode 644.
 - use `git add *`, **NEVER**.
 
 
+### Best Practices: Pull Requests
+For Pull Requests, there are two kind of best practices, related to the Pull Request itself, and to the Pull Request
+Review, that we established in [@ipa320](https://github.com/ipa320).
+
+- Pull Request
+  - keep the PR small (<100 changed files, <1000 changed lines of code)
```

Comment:
<100LOC is going to be interesting. I think we'll cross this bound pretty often.
Except for parameter adaptations or remappings, every new feature will probably cross this.

I'd say about 300 makes more sense. What does the rest of you guys think? @ipa-nhg @ipa-fxm 
And then add an "exception" for new packages, as they will always violate this...